### DISCUSSION TITLE
[Question] Schedule for final exam

### DISCUSSION THREAD

**Discussion by yusungsim** [prosyslab-classroom/cs424-program-reasoning#335] [Q&A]
I was curious that when will be the final exam.
Will it be the normal offline exam? If so, what is the scheduled date and time of the exam?

**Comment by KihongHeo** [comment]
Hi.

- Dec 13 9:00
- Open book

The details will be announced later.

**Comment by yusungsim** [comment]
Thank you for reply :)
